packages/contracts-bedrock/scripts/L2Genesis.s.sol

```diff
@@ -249,6 +252,36 @@ contract L2Genesis is Deployer {
         _setImplementationCode(Predeploys.L2_TO_L1_MESSAGE_PASSER);
     }
 
+    function setSoulGasToken() public {
+        address impl = _setImplementationCode(Predeploys.SOUL_GAS_TOKEN);
+
+        if (Constants.IS_SOUL_BACKED_BY_NATIVE) {
+            SoulGasToken(payable(impl)).initialize({
+                name_: "SoulQKC",
```

Comment:
I think we could use SoulGasToken for all the names.

Reply:
Fixed.